layers/cmd/export_csv.go

```diff
@@ -24,6 +24,34 @@ import (
 	lscsv "github.com/cloudprivacylabs/lsa/pkg/csv"
 )
 
+type ExportCSV struct {
+	Input string
+	Spec  string
+}
```

Comment:
Let's have spec point to a spec file. Alternatively, the spec can be inlined here.
type ExportCSV {
   SpecFile string
   Spec ...
}